agent/acs/model/ecsacs/api.go

```diff
@@ -413,6 +413,24 @@ func (s ECRAuthData) GoString() string {
 	return s.String()
 }
 
+type EFSVolumeConfiguration struct {
+	_ struct{} `type:"structure"`
+
+	Filesystem *string `locationName:"filesystem" type:"string"`
```

Comment:
`FilesystemID` makes more sense

Reply:
👍

Reply:
I've changed it to `FilesystemId` as this is auto-generated code that depends on the ACS model, which will have the `d` lowercase.